modules/backup-and-restore/pages/cross-cluster-backup.adoc

```diff
@@ -7,6 +7,35 @@ This page walks you through the process of restoring a cluster from a backup tha
 Restoring is an off-line operation.
 Your cluster is offline during the restore process.
 
+NOTE: Since v3.9.3, it is no longer required to create a backup for a backups metadata or upload the metadata to a new cluster.
+A backup’s metadata is now stored as a part of a backup's files.
+
+Simply run:
+
+[source,console,]
+--
+gadmin backup restore <backup_tag>
+--
+
+Just like restoring a backup in its original cluster.
+
+For local backup, the restore flow will try to find a metadata file under the backup path:
+
+[source,console]
+--
+/path/to/{System.Backup.Local.Path}/<backup_tag>/metadata
+--
+
+For the S3 backup, the restore flow will try to find an s3 object:
+
+[source, console]
+--
+<backup_tag>_metadata
+--
+
+From the Bucket `System.Backup.S3.BucketName`.
+
+IMPORTANT: For all previous versions please follow the steps below.
```

Comment:
for the version 3.9.2+, we still need step 2.2 https://docs.tigergraph.com/tigergraph-server/current/backup-and-restore/cross-cluster-backup#_copy_data_backup_to_new_cluster.

FYI:
     
compared to the previous version, what we can simplify are
1. there is no need to export the backup's metadata manually since the backup files have contained it.
2. no need to specify the metadata for the restore command, since it uses the metadata, at the backup files position, as default.

but we still support the old ways.

Reply:
Thank you fro the clarifications. I appreciate you. Changed this with latest commit.